lib/rails_admin/engine.rb

```diff
@@ -7,9 +7,28 @@
 require 'rails'
 require 'rails_admin'
 require 'remotipart'
-require 'safe_yaml/load'
+require 'yaml'
 
 module RailsAdmin
+  # Backwards-compatible RailsAdmin::SafeYAML when SafeYAML isn't available
+  begin
+    require 'safe_yaml/load'
+  rescue LoadError
+    if YAML.respond_to?(:safe_load)
+      SafeYAML = Module.new do
+        def self.load(yaml)
+          YAML.safe_load(yaml)
+        end
+      end
+    else
+      warn "Safe-loading of YAML is not available. Please install 'safe_yaml' or install Psych 2.0+"
```

Comment:
An error should be raised here, not warning. We shouldn't let users choose unsafe state.